src/App.tsx

```diff
@@ -73,6 +75,13 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
 
+	const flowTheme =
```

Comment:
We already have this code in the dynamicSdk branch
Do we really need it also here?

Reply:
I think it's important in case someone clones the repo, and expects to follow our docs to be able to use this functionality.

Reply:
Bot sure I agree, but anyhow the code. should be the same or we will never be able to merge back once we get rid of dynamic SDK